src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsFetchingSpec.groovy

```diff
@@ -1733,6 +1735,33 @@ class PriceFloorsFetchingSpec extends PriceFloorsBaseSpec {
         }
     }
 
+    def "PBS should populate floors enabled = true when floors skip rate 100"() {
```

Comment:
Please create a test for when floors are disabled and skip rate is 100 if there is no one with the same conditions.